src/test/java/com/connexta/search/common/SearchITests.java

```diff
@@ -429,4 +409,28 @@ public URL solrUrl() throws MalformedURLException {
           "/solr");
     }
   }
+
+  private HttpEntity createIndexRequest(final String fileString) throws IOException {
+    // TODO replace with request class from api dependency
+    final InputStream metadataInputStream = IOUtils.toInputStream(fileString, "UTF-8");
+    final MultiValueMap<String, Object> requestBody = new LinkedMultiValueMap<>();
+    requestBody.add(
+        "file",
+        new InputStreamResource(metadataInputStream) {
+
+          @Override
+          public long contentLength() throws IOException {
+            return metadataInputStream.available();
+          }
+
+          @Override
+          public String getFilename() {
+            // The extension of this filename is used to get the ContentType of the file.
+            return "ignored.json";
+          }
+        });
+    final HttpHeaders httpHeaders = new HttpHeaders();
+    httpHeaders.set(IndexController.ACCEPT_VERSION_HEADER_NAME, indexApiVersion);
```

Comment:
✏️ Since this is the only use of IndexController just static import ACCEPT_VERSION_HEADER_NAME

Reply:
The reason I chose to keep it this way is because we probably will add an Accept-Version to the QueryApi, and I wanted to be explicit that this field should only be used for the IndexApi. An alternative would be naming the field INDEX_API_ACCEPT_VERSION_HEADER_NAME.